flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -96,7 +111,11 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
   final RouteNameExtractor? _routeNameExtractor;
   final AdditionalInfoExtractor? _additionalInfoProvider;
   final SentryNative? _native;
-  final TimeToDisplayTracker? _timeToDisplayTracker;
+  late final TimeToDisplayTracker? _timeToDisplayTracker;
+
+  @internal
+  static TimeToDisplayTracker? get timeToDisplayTracker =>
+      SentryNavigatorObserver()._timeToDisplayTracker;
```

Comment:
```suggestion
  static TimeToDisplayTracker? _timeToDisplayTracker;

  @internal
  static TimeToDisplayTracker? get timeToDisplayTracker => _timeToDisplayTracker;
```
This way we avoid creating an instance of the observer just to access the tracker